.rubocop.yml

```diff
@@ -0,0 +1,16 @@
+Documentation:
+  Enabled: false
+Style/StringLiterals:
+  EnforcedStyle: double_quotes
```

Comment:
I agree with about 99.5% of what Thoughtbot says. :stuck_out_tongue: Sort of surprised at this one, considering they’re a vim shop. For me, the extra keystroke (Shift) for a double quote suggests an intentially “special” string, matching up with the thought of declaring it interpolated. This is a reasonable/negligible overhead imo, for the added benefits, once you’re used to the convention.

But anyhow, I digress – if everyone else on the team prefers double quotes then we should definitely go with that!